README.md

```diff
@@ -1,6 +1,6 @@
 # Sargon
 
-[![codecov](https://codecov.io/github/radixdlt/Sargon/graph/badge.svg?token=EQYDU0XPMX)](https://codecov.io/github/radixdlt/Sargon)
+[![codecov](https://codecov.io/gh/radixdlt/Sargon/graph/badge.svg?token=8QPKIUSAQD)](https://codecov.io/github/radixdlt/Sargon)
```

Comment:
we were using the wrong token, but it seems it did not matter. I still cannot produce badges for specific flags: `kotlin` / `swift` / `rust`, but this PR adds a `codecov.yml` file which *might* fix that (Im guessing here...).